easybuild/easyconfigs/a/ANTLR/ANTLR-2.7.7-CrayGNU-17.08-Python-2.7.13.eb

```diff
@@ -0,0 +1,34 @@
+# contributed by Luca Marsella (CSCS)
+easyblock = 'ConfigureMake'
+
+name = 'ANTLR'
+version = "2.7.7"
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.antlr.org/'
+description = """ANTLR, ANother Tool for Language Recognition, (formerly PCCTS)
+ is a language tool that provides a framework for constructing recognizers,
+ compilers, and translators from grammatical descriptions containing
+ Java, C#, C++, or Python actions."""
+
+toolchain = {'name': 'CrayGNU', 'version': '17.08'}
+
+source_urls = ['http://www.antlr2.org/download/']
+sources = [SOURCELOWER_TAR_GZ]
+
+patches = ['%(name)s-%(version)s_includes.patch']
```

Comment:
can you push the patch file ?

Reply:
Surprisingly, there is no patch file. I have not modified this line; it was already in the previous recipe...